main.py

```diff
@@ -139,4 +149,14 @@ def write_done_file(dated_subfolder):
     print(f"Uploading done file ...")
     s3.Object(s3_bucket, f"{dated_subfolder}/done").put(Body="")
 
-main()
+
+def lz4(src):
+    # See also idseq_dag/steps/generate_lz4.py
+    dst = src + '.lz4'
+    print(f"Compressing {src} to lz4 ...")
+    command_execute(f'lz4 -9 -f {src} {dst}')
```

Comment:
not sure how long this took you, but when I generated some manually, I found that the "-9" seemed to take quite a long time for not so much benefit